crates/matrix-sdk-sled/src/state_store/mod.rs

```diff
@@ -102,53 +102,59 @@ impl From<SledStoreError> for StoreError {
     }
 }
 
-const ACCOUNT_DATA: &str = "account-data";
-const CUSTOM: &str = "custom";
-const SYNC_TOKEN: &str = "sync_token";
-const DISPLAY_NAME: &str = "display-name";
-const INVITED_USER_ID: &str = "invited-user-id";
-const JOINED_USER_ID: &str = "joined-user-id";
-const MEDIA: &str = "media";
-const MEMBER: &str = "member";
-const PRESENCE: &str = "presence";
-const PROFILE: &str = "profile";
-const ROOM_ACCOUNT_DATA: &str = "room-account-data";
-const ROOM_EVENT_RECEIPT: &str = "room-event-receipt";
-const ROOM_INFO: &str = "room-info";
-const ROOM_STATE: &str = "room-state";
-const ROOM_USER_RECEIPT: &str = "room-user-receipt";
-const ROOM: &str = "room";
-const SESSION: &str = "session";
-const STRIPPED_INVITED_USER_ID: &str = "stripped-invited-user-id";
-const STRIPPED_JOINED_USER_ID: &str = "stripped-joined-user-id";
-const STRIPPED_ROOM_INFO: &str = "stripped-room-info";
-const STRIPPED_ROOM_MEMBER: &str = "stripped-room-member";
-const STRIPPED_ROOM_STATE: &str = "stripped-room-state";
-
-const ALL_DB_STORES: &[&str] = &[
-    ACCOUNT_DATA,
-    SYNC_TOKEN,
-    DISPLAY_NAME,
-    INVITED_USER_ID,
-    JOINED_USER_ID,
-    MEDIA,
-    MEMBER,
-    PRESENCE,
-    PROFILE,
-    ROOM_ACCOUNT_DATA,
-    ROOM_EVENT_RECEIPT,
-    ROOM_INFO,
-    ROOM_STATE,
-    ROOM_USER_RECEIPT,
-    ROOM,
-    SESSION,
-    STRIPPED_INVITED_USER_ID,
-    STRIPPED_JOINED_USER_ID,
-    STRIPPED_ROOM_INFO,
-    STRIPPED_ROOM_MEMBER,
-    STRIPPED_ROOM_STATE,
-    CUSTOM,
-];
+#[allow(non_snake_case)]
+mod KEYS {
```

Comment:
Can we keep module names lowercase in sled? We should also change indexeddb, but I can take care of that in a separate PR if you prefer.